recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
+
+build:
+  number: 0
+  script: python setup.py install
+
+
+requirements:
+  build:
+    - python
+    - param
+    - numpy
+    - setuptools
+  run:
+    - python
+    - param
+    - numpy
+    - matplotlib
+    - bokeh
+    - jupyter
+    - notebook
+    - ipython
+    - lancet
+
+test:
+  imports:
+    - holoviews
+
+about:
+  home: www.holoviews.org
+  summary: Stop plotting your data - annotate your data and let it visualize itself.
+  license: BSD
```

Comment:
It is the 3-Clause. If the right value ``BSD3``?

https://github.com/ioam/holoviews/blob/master/LICENSE.txt